react-xmasonry/build.boot

```diff
@@ -19,7 +19,7 @@
 (deftask package []
   (comp
    (download :url (format "https://github.com/ZitRos/react-xmasonry/archive/master.zip")
-             :checksum "E6B128183607A37A561E58E5D913CF4C"
+             :checksum "D9F26537EB811BC06717D08A93E58235"
```

Comment:
The url should use `+lib-version+` and the checksum should stay the same if version is not updated. Now the master branch points to version 2.0.1 instead of 1!

```
(format "https://unpkg.com/react-xmasonry@%s/dist/index.js" +lib-version)
```

Or something, as the package doesn't use git tags.